static/js/TopicPage.jsx

```diff
@@ -351,13 +352,30 @@ const TopicHeader = ({ topic, topicData, multiPanel, isCat, setNavTopic, openDis
         <div>
           <div className="sectionTitleText authorIndexTitle"><InterfaceText>Works on Sefaria</InterfaceText></div>
           <div className="authorIndexList">
-            {topicData.indexes.map(({text, url}) => <SimpleLinkedBlock key={url} {...text} url={url} classes="authorIndex" />)}
+            {topicData.indexes.map(({url, title, description}) => <AuthorDisplayItem key={url} url={url} enTitle={title.en} heTitle={title.he} enDesc={description.en} heDesc={description.he}/>)}
           </div>
         </div>
        : null}
     </div>
 );}
 
+const AuthorDisplayItem = ({url, enTitle, heTitle, enDesc, heDesc}) => {
+  const classes = classNames({ navBlockTitle: 1 });
+  return (
+    <div className="navBlockAuthorPage" >
```

Comment:
This code is unfortunately duplicated in many places in our code with slight variations. I think it's out of scope for this project but something to be careful about.